templates/minimal/README.md

```diff
@@ -45,65 +69,155 @@ git clone https://github.com/paritytech/polkadot-sdk-minimal-template.git minima
+[on crates.io](https://crates.io/crates/polkadot-omni-node).
+
+#### Install `polkadot-omni-node`
+
+```sh
+cargo install polkadot-omni-node
+```
+
+#### Build `minimal-template-runtime`
 
-🔨 Use the following command to build the node without launching it:
+```sh
+cargo build -p minimal-template-runtime --release
+```
+
+#### Install `staging-chain-spec-builder`
 
 ```sh
-cargo build --release
+cargo install staging-chain-spec-builder
 ```
 
-🐳 Alternatively, build the docker image:
+#### Use chain-spec-builder to generate the chain_spec.json file
+
+```sh
+chain-spec-builder create --relay-chain "dev" --para-id 1000 --runtime \
+    <target/release/wbuild/path/to/minimal-template-runtime.wasm> named-preset development
+```
+
+**Note**: the `relay-chain` and `para-id` flags are extra bits of information required to
+configure the node for the case of representing a parachain that is connected to a relay chain.
+They are not relevant to minimal template business logic, but they are mandatory information for
+Omni Node, nonetheless.
+
+#### Run Omni Node
+
+Start Omni Node with manual seal (3 seconds block times) and minimal template runtime based
+chain spec:
+
+```sh
+polkadot-omni-node --chain <path/to/chain_spec.json> --dev-block-time 3000 --tmp
+```
+
+### Minimal Template Node
+
+#### Build both node & runtime
+
+```sh
+cargo build --workspace --release
+```
+
+🐳 Alternatively, build the docker image which builds all the workspace members,
+and has as entry point the node binary:
 
 ```sh
 docker build . -t polkadot-sdk-minimal-template
 ```
 
-### Single-Node Development Chain
+#### Start the `minimal-template-node`
 
-👤 The following command starts a single-node development chain:
+The `minimal-template-node` has a compile time dependency on `minimal-template-runtime`. The
+`minimal-template-runtime` exposes a constant that points to the runtime WASM blob, which is needed
+by the node to be able to create a chain spec, which will be used further in its setup, before it
+starts fully.
 
 ```sh
-./target/release/minimal-template-node --dev
+<target/release/path/to/minimal-template-node> 
+# or via docker
+docker run --rm polkadot-sdk-minimal-template
+```
+
+### Zombienet with Omni Node
+
+#### Install `zombienet`
+
+We can install `zombienet` as described [here](https://paritytech.github.io/zombienet/install.html#installation),
+and `zombienet-omni-node.toml` contains the network specification we want to start.
+
+#### Update `zombienet-omni-node` with a valid chain spec path
 
-# docker version:
-docker run --rm polkadot-sdk-minimal-template --dev
+Before starting the network with zombienet we must update the network specification
+dawith a valid chain spec path. If we need to generate one, we can look up at the previous
+section for chain spec creation [here](#use-chain-spec-builder-to-generate-the-chain-spec-json-file).
+
+Then make the changes in the network specification like so:
+
+```toml
+# ...
+chain = "dev"
```

Comment:
I don't get why the `zombienet-multi-node.toml` provided is not "ready to use", and has to be edited?

Reply:
How about shipping templates with the `development` chain-spec then? The json file is there, it is already linked in `zombienet.toml`, and it can be updated if need be? 

The goal for a template should be to be run-able with minimum effort. 

Can be a follow-up

Reply:
Actually, I think this is a solid plan: 

Each template has 2 "flows":

1. Build the files and settings step by step (what we have now)
2. `quickstart.sh`: One command that runs the whole thing

So it is easy to keep this centered around flow 1 in this PR, and add flow 2 in a follow-up.